### ISSUE TITLE
Adding a second Linear.app server invalidates API for the first one (after restart)

### ISSUE THREAD

**Issue by failedguidedog** [mayankmkh/linear-app-plugin#155]
I filed [an issue](https://youtrack.jetbrains.com/issue/IDEA-313033) on PyCharm's issue tracker by mistake, not realizing I had this plugin installed.

Steps to reproduce:

1. Tools -> Tasks & Contexts -> Configure servers -> Add server -> Linear
2. Configure Team ID and API key
3. Click Test button and confirm the connection is successful
4. Click OK (closing the modal)
5. Repeat steps 1 thrugh 4
6. Exit PyCharm
7. Start PyCharm
8. Tools -> Tasks & Contexts -> Configure servers
9. Confirm that the first server now has empty API key field and the Test button is grayed out.

**Comment by jornetsimon** [comment]
Yes, it is very inconvenient when working across multiple teams 🙁
